app/helpers/requests_helper.rb

```diff
@@ -0,0 +1,13 @@
+module RequestsHelper
+  def quota_display(partner)
+    return "" if partner.quota.blank?
+
+    "(#{partner.quota})"
+  end
+
+  def quota_column_class(total, partner)
+    return "" if partner.quota.blank? || total < partner.quota
+
+    "text-danger font-weight-bold"
```

Comment:
Is red text appropriate here? The convention in the app is that red text is used for errors and to indicate negative quantities (sometimes when something is "alert! this is probably not ok!")

If you're wanting to accent the text I would suggest a neutral hue like blue or grey, or to display it with text formatting like italics or bold (or make the main number bold and this one plain?).

If this number is displaying because there is a legitimate alert that requires user attention, then red is appropriate :+1:

Reply:
@armahillo That's based on Gia's mockup -- the idea is that we want requests that are outside the distribution limit to be flagged in a very visible way. The diaper bank of the ozarks said they get over 70 requests a month so being able to quickly scroll through a big list and have them stand out would be appreciated.

Reply:
Good point. I'll see what I can do about that.

Reply:
Hey there -- sorry I'am late to the show! Here is some recommendations on approaching this:
- Apply a background-color on rows that need to stand out.
- Separate Request Limit in its own column & possibly another column called 'Over Quota By' which would indicate how much over it is. I suspect requesting 1 more than the request limit is okay in cases. 

Here is a quick mock-up of just what it looks like with the background-color set to darkorange:
![Screen Shot 2020-09-26 at 11 51 37 AM](https://user-images.githubusercontent.com/11335191/94345941-ed7e5400-ffee-11ea-9c2b-13f7971c4d8e.png)

@designbygia I know this is from your mock-up. Let's talk about this :)